### PR TITLE
Upstash Kafka metrics and dashboards added

### PR DESCRIPTION
**Pull request by WoodProgrammer**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: upstash/metadata.csv
Comment (bgoldberg122) [inline upstash/metadata.csv:1]: It's not so straightforward to update metric names, is the name update required here?

Comment (WoodProgrammer) [reply upstash/metadata.csv:1]: Hello @bgoldberg122 I handled this issue and added new metrics instead of adjust existing ones.

Comment (WoodProgrammer) [reply upstash/metadata.csv:1]: Could you please check this again ?

Comment (WoodProgrammer) [reply upstash/metadata.csv:1]: Hello is there any progress ?

Comment (bgoldberg122) [reply upstash/metadata.csv:1]: Hi @WoodProgrammer, sorry for the delay, I was out of office last week, the new metric names look good

**Comment by bgoldberg122** [comment]
Hi @WoodProgrammer, sorry for the delay, I was out of office last week. Could you please add a new `CHANGELOG` entry for this update?

**Comment by WoodProgrammer** [comment]
> Hi @WoodProgrammer, sorry for the delay, I was out of office last week. Could you please add a new `CHANGELOG` entry for this update?

Changelog added 👍

**Review by emarsha94 — APPROVED** [review]

**Review by bgoldberg122 — APPROVED** [review]

**Pull request merged**